diff --git a/dist/index.cjs b/dist/index.cjs
index 03ed29130b5f3fac62973399ab3946fce44162e3..094f74f9cd84d4ff6ad193f9e97f499ddc977422 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -98,7 +98,7 @@ const presetTheme = (options) => {
               if (kind === "dark" || kind === "light")
                 return cleanCode;
             }
-            return `${kind === "light" ? ":root" : `.${kind}`}${targetCSS || ""}`;
+            return `${kind === "light" ? "body" : `.${kind}`}${targetCSS || ""}`;
           });
         }
       }
diff --git a/dist/index.mjs b/dist/index.mjs
index b64a0d48638f9476c5e40b27865670aee9ce8abb..b3631fcc711071dd791b9c591bfaf74e2562bd0d 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -94,7 +94,7 @@ const presetTheme = (options) => {
               if (kind === "dark" || kind === "light")
                 return cleanCode;
             }
-            return `${kind === "light" ? ":root" : `.${kind}`}${targetCSS || ""}`;
+            return `${kind === "light" ? "body" : `.${kind}`}${targetCSS || ""}`;
           });
         }
       }